src/main/java/com/google/starfish/services/GenericService.java

```diff
@@ -0,0 +1,57 @@
+package com.google.starfish.services;
+
+import java.sql.ResultSet;
+import javax.sql.DataSource;
+import java.sql.PreparedStatement;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+/** Enum that holds starfishDB sql tables as strings */
+enum Table {
+  USERS("users"), NOTES("notes"), LABELS("labels"), FAVORITE_NOTES("favorite_notes"), MISC_LABELS("misc_note_labels");
+
+  private String sqlTable;
+
+  public String getSqlTable() {
+    return this.sqlTable;
+  }
+
+  private Table(String sqlTable) {
+    this.sqlTable = sqlTable;
+  }
+}
+
+/** Generic Service class that provides the functionality to get and delete row by id in any table */
+public class GenericService {
+
+  /** Gets row by table and id from sql */
+  public ResultSet getById(DataSource pool, long id, Table sqlTable) throws SQLException {
```

Comment:
nit: I think being a little more verbose like `getRowById` or even just `getRow` (with param name maybe named `rowId` to be a little more explicit) could be better since I might think this is a method for getting a table by ID

Reply:
Good point, I really like `getRowById`, I think that's still succinct but more descriptive for sure. Will also change param `id` to `rowId`.